modules/javafx.controls/src/android/java/javafx/scene/control/skin/TextFieldSkinAndroid.java

```diff
@@ -71,6 +74,15 @@ public TextFieldSkinAndroid(final TextField textField) {
      *                                                                         *
      **************************************************************************/
 
+    /** {@inheritDoc} */
+    @Override protected String maskText(String txt) {
+        if (getSkinnable() instanceof PasswordField) {
+            return String.valueOf(BULLET).repeat(txt.length());
+        } else {
+            return txt;
```

Comment:
Should this call `return super.maskText();` in case the implementation of the superclass method ever changes? Either is OK with me.

Reply:
I think that makes sense, even if it causes a double check of getSkinnable(), but that seems negligible. I'll update it